### ISSUE TITLE
Module Db with postegre

### ISSUE THREAD

**Issue by xero88** [Codeception/Codeception#2353]
Hi,

I'm trying to assert some data in db (postegr db), but I get this error
```
$I->seeInDatabase('schema1.source', array('name' => 'print'));
```

Error : 

```
[PDOException] SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "schema1.source" does not exist
LINE 1: select count(*) from "schema1.source" where schema1...
```

Here my configuration : 
```
modules:
    config:
        Db:
            dsn: 'pgsql:host=alnus;port=5432;dbname=database1'
            user: 'gbo'
            password: 'mypassword'
            dump: tests/_data/dump.sql
            populate: false
```
My table "source" is in a schema "schema1" in the db "database1"

Thank you !

**Comment by Naktibalda** [comment]
Which version of Codeception are you using? This issue was fixed in 2.0.8

**Comment by xero88** [comment]
I'm using version 2.1.0

**Comment by Naktibalda** [comment]
Upgrade to 2.1.2

**Comment by xero88** [comment]
Solved by upgrading to 2.1.2 =)

Thank you!
